communities.json

```diff
@@ -679,6 +679,13 @@
       "quote": "Discord both strengthened our community and dramatically improved discussion amongst the developers.",
       "inviteCode": "G7Qnnhy",
       "githubUrl": "https://github.com/nestjs"
+    },
+    {
+      "logo": "pupil_labs.svg",
+      "title": "Pupil Labs",
+      "quote": "Pupil Labs builds state of the art eye tracking hardware and software. Our team and community uses Discord to connect, write code, share research, and exchange ideas!",
+      "inviteCode": "gKmmGqy",
```

Comment:
```suggestion
      "inviteCode": "6MtAMU",
```

Reply:
Should be `dc2FYDQ`

Reply:
Not necessary to change this. Can keep as is in the original commit.